ddtrace/mocktracer/mockspan_test.go

```diff
@@ -65,6 +66,20 @@ func TestNewSpan(t *testing.T) {
 		assert.Equal(uint64(2), s.context.traceID)
 		assert.Equal(baggage, s.context.baggage)
 	})
+	t.Run("custom_source", func(t *testing.T) {
+		tr := new(mocktracer)
+		parentctx := &spanContext{spanID: 1, traceID: 2}
+		tags := make(map[string]interface{})
+		tags[ext.Component] = "sourceName"
```

Comment:
If you change the name of the mockspan struct field from `source` to something else (like `integration`), would be good to change the value here (and on line 81) to match this change.